runners/java-fn-execution/src/test/java/org/apache/beam/runners/fnexecution/control/RemoteExecutionTest.java

```diff
@@ -152,6 +159,7 @@ public void tearDown() throws Exception {
     controlClient.close();
     sdkHarnessExecutor.shutdownNow();
     serverExecutor.shutdownNow();
+    sdkHarnessExecutorFuture.get();
```

Comment:
This line is failing the tests due to an `InterruptedException` 
  https://scans.gradle.com/s/nihiogx5epq4i/tests/failed

I can reproduce this building your branch with
```
./gradlew :beam-runners-java-fn-execution:test
```